README.md

```diff
@@ -7,3 +7,26 @@ This the place to suggest large enhancements that may impact several repositorie
 * Additions/changes to the message spec
 * API design that is consumed and produced in several contexts (e.g. the notebook kernel API)
 * The notebook format
+
+## Enhancement Proposal Submission Guidlines
+
+When submitting a enhancement proposal, please include the following information in your submission.
+
+1. The problem that this enhancement addresses. If possible include code or anecodotes to describe this problem to readers.
+2. A brief (1-2 sentences) overview of the enhancement you are prposing. If possible include hypothetical code sample to describe how the solution would work to readers.
```

Comment:
I KNOW HOW TO TYPE I SWEAR I DON'T HAVE FAT FINGERS. 

I'll fix these.

![giphy-2](https://cloud.githubusercontent.com/assets/1857993/9536914/c4452e44-4ce6-11e5-9d9e-2898ef3659dd.gif)